apps/web/pages/auth/logout.tsx

```diff
@@ -1,6 +1,6 @@
 import type { GetServerSidePropsContext } from "next";
-import { signOut, useSession } from "next-auth/react";
-import { useRouter } from "next/navigation";
+import { destroy, useSession } from "next-auth/react";
+import { useRouter } from "next/router";
```

Comment:
We're deprecating importing from "next/router" as we're planning to migrate to app router and it is incompatible.